packages/cli/src/commands/apm_cmds/util/runPrepareForPublishTask.js

```diff
@@ -1,21 +1,29 @@
 import tmp from 'tmp-promise'
 import TaskList from 'listr'
-import { APM_INITIAL_VERSIONS, apmPublishVersionIntent } from '@aragon/toolkit'
+import path from 'path'
+import fs from 'fs'
+import { promisify } from 'util'
+import { readJsonSync } from 'fs-extra'
+import {
+  APM_INITIAL_VERSIONS,
+  apmPublishVersionIntent,
+  generateApplicationArtifact,
+} from '@aragon/toolkit'
 //
 import listrOpts from '../../../helpers/listr-options'
 
 import askToConfirm from '../../../lib/publish/askToConfirm'
 import { prepareFilesForPublishing } from '../../../lib/publish/preprareFiles'
 import {
   changeManifestForHttpServedFrom,
-  generateApplicationArtifact,
   writeApplicationArtifact,
   flattenCodeFileExists,
   generateFlattenedCode,
   artifactExists,
   checkIfNewArticatIsIdentical,
   copyCurrentApplicationArtifacts,
 } from '../../../lib/apm/generateArtifact'
+const readFile = promisify(fs.readFile)
```

Comment:
We can now use `fs.promises` since we don't support nodejs 8 anymore.

```suggestion
const readFile = fs.promises.readFile
```